perfdash/README.md

```diff
@@ -0,0 +1,22 @@
+﻿# Kubernetes Perfdash
+
+Kubernetes Perfdash (performance dashboard) is a web UI that collects and displays performance metrics. Performance metrics are created based on performance test results for different nodes numbers, platform types and platform versions.
+
+Perfdash is available at http://perf-dash.k8s.io/
+
+## Supported metrics
+
+* Responsiveness
+* Resources
+* PodStartup
+* TestPhaseTimer
+* RequestCount
+* RequestCountByClient
+
+Metrics above are available for all kinds of tests divided into load and density subtypes.
+
+## Application server
+Application server is hosted on grc.io/k8s-mungegithub project cluster.
+
+## Application images
+Images are stored in gcr.io/k8s-staging project container registry.
```

Comment:
As far I know, those should be two different projects.
Anyway, there should be gcr.io/k8s-image-staging instead of gcr.io/k8s-staging. I need to correct it.